packages/cli/lib/commands/create.js

```diff
@@ -78,13 +78,13 @@ const options = [
 ];
 
 // Formulate Questions if `create` args are missing
-function requestParams(argv, templates) {
+function requestParams(repo, dest, argv, templates) {
```

Comment:
Is there a reason to drag in `dest` too?

Reply:
https://github.com/preactjs/preact-cli/blob/0ccce89fc2ed6e9367d5889f9c16f2f3cd89f988/packages/cli/lib/commands/create.js#L99

There isn't a `--dest` flag.

Reply:
Whoops, totally right. Forgot we used 'name' instead.